<launch>

  <!-- <node name="stitched_frame_publisher" pkg="showmap" type="calibration.py" output="screen">
  </node> -->

  <node name="video_stream0" pkg="video_stream" type="trace" output="screen">
    <param name="video_device" value="camera_0" />
  </node>

  <node name="video_stream1" pkg="video_stream" type="trace" output="screen">
    <param name="video_device" value="camera_1" />
  </node>

  <node name="video_stream2" pkg="video_stream" type="trace" output="screen">
    <param name="video_device" value="camera_2" />
  </node>

  <node name="video_stream3" pkg="video_stream" type="trace" output="screen">
    <param name="video_device" value="camera_3" />
  </node>

  <node name="video_stream4" pkg="video_stream" type="trace" output="screen">
    <param name="video_device" value="camera_4" />
  </node>

  <node name="video_stream5" pkg="video_stream" type="trace" output="screen">
    <param name="video_device" value="camera_5" />
  </node>

  <node name="web_video_server01" pkg="web_video_server" type="web_video_server" output="screen">
    <param name="address" value="0.0.0.0" />
    <param name="port" value ="8000" />
    <param name="video_encoder" value="mjpeg" />
  </node>

  <!-- <node name="stitched_frame" pkg="video_stream" type="record" output="screen">
  </node> -->

  <!-- <node name="merged_frame" pkg="video_stream" type="merge" output="screen">
  </node> -->

  <node name="multi" pkg="video_stream" type="multi_band" output="screen">
    <param name="partial_1_2" value="8" />
    <param name="partial_2_3" value="7" />
    <param name="partial_3_4" value="6" />
    <param name="partial_4_5" value="7" />
    <param name="partial_5_6" value="6" />
    <param name="partial_6_1" value="6" />
  </node>

</launch>